### DISCUSSION TITLE
Svelte & HMR support

### DISCUSSION THREAD

**Discussion by braebo** [luckyframework/lucky#1410] [Feature requests & Ideas]
Are there any plans support Svelte?  I think It would be a killer combo!  Also wondering what the story is on HMR support when developing with Lucky.

**Comment by jwoertink** [comment]
Hey @FractalHQ! I've never used Svelte before, but what all would you need in order to support it? Is there something specific in Lucky that prevents you from using it? Also, what is "HMR"? I'm not familiar with this. I assume you don't mean the diet plan thing 😛
